test/unit/test_sap_cli_gcts_cmd_messages.py

```
@@ -25,7 +25,17 @@
-class TestgCTSRepoMessages(PatcherTestCase, ConsoleOutputTestCase):
+class TestgCTSRepoMessages(ConsoleOutputTestCase, PatcherTestCase):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        PatcherTestCase.__init__(self)
+
+    def tearDown(self):
+        try:
+            PatcherTestCase.unpatch_all(self)
+        finally:
+           super().tearDown()
```

Flake8 (7.3.0)

[error] indentation is not a multiple of 4

(E111)